app/views/cardview.xml

```diff
@@ -2,13 +2,13 @@
 	<Tab title="CardView">
 		<Window title="CardView">
 			<ScrollView layout="vertical">
-				
+
 				<!-- FIXME: with ns styles is not working with Alloy 1.7.18 -->
-				<CardView>
+				<CardView class="card" ns="Ti.UI.Android" >
```

Comment:
The sample requires Alloy 1.7.20 or later and I've added a JMK to check that at compile now and also a note in the readme.